sql: TestDistSQLRangeCachesIntegrationTest failed

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174289&tab=buildLog) on [master@44c362600dbed9d58be8daf64558543745698736](https://github.com/cockroachdb/cockroach/commits/44c362600dbed9d58be8daf64558543745698736):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest791214954
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.38s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest791214954
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174505&tab=buildLog) on [master@ed56390d4f28f7d7b26ce047bcb799afb337b537](https://github.com/cockroachdb/cockroach/commits/ed56390d4f28f7d7b26ce047bcb799afb337b537):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest742113835
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.21s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest742113835
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174486&tab=buildLog) on [master@09bb6460c0ed1a882db95472e2d962dd57a80aaf](https://github.com/cockroachdb/cockroach/commits/09bb6460c0ed1a882db95472e2d962dd57a80aaf):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest122969999
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.40s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest122969999
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174851&tab=buildLog) on [master@239805ade7d2e2e2628266fefc818ac60dcf74dc](https://github.com/cockroachdb/cockroach/commits/239805ade7d2e2e2628266fefc818ac60dcf74dc):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest032482438
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.30s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest032482438
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174988&tab=buildLog) on [master@be8c1a17b940ecc56dc3521ab8264f662c88072c](https://github.com/cockroachdb/cockroach/commits/be8c1a17b940ecc56dc3521ab8264f662c88072c):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest244124105
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.26s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest244124105
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}
W200812 18:02:43.994551 21584 kv/txn.go:618  [n4,client=127.0.0.1:45280,hostssl,user=root] failure aborting transaction: node unavailable; try another peer; abort caused by: connExecutor closing

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2175074&tab=buildLog) on [master@9803ab8458d2b8b1b1abc63f32a49d216b126137](https://github.com/cockroachdb/cockroach/commits/9803ab8458d2b8b1b1abc63f32a49d216b126137):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest659254829
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.20s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest659254829
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2175218&tab=buildLog) on [master@89208ec8b5350fd34036b559a258403e820deda6](https://github.com/cockroachdb/cockroach/commits/89208ec8b5350fd34036b559a258403e820deda6):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest504929075
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.63s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest504929075
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2175492&tab=buildLog) on [master@52b8eb1b1c90d1ff9c8faae8c80d45b7de56d350](https://github.com/cockroachdb/cockroach/commits/52b8eb1b1c90d1ff9c8faae8c80d45b7de56d350):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest857377982
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.57s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest857377982
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDistSQLRangeCachesIntegrationTest failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2175690&tab=buildLog) on [master@28c0337df4b695b6eba09639ca312a92348b9294](https://github.com/cockroachdb/cockroach/commits/28c0337df4b695b6eba09639ca312a92348b9294):

```
=== RUN   TestDistSQLRangeCachesIntegrationTest
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest130916110
--- FAIL: TestDistSQLRangeCachesIntegrationTest (4.94s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestDistSQLRangeCachesIntegrationTest130916110
    test_log_scope.go:63: use -show-logs to present logs inline
    distsql_physical_planner_test.go:419: expected json to contain "nodeNames":["1","4"], but json is: {"sql":"SELECT json FROM [EXPLAIN (DISTSQL) SELECT count(1) FROM \"left\" INNER JOIN \"right\" USING (num)]","nodeNames":["1","2","3","4"],"processors":[{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/0","details":["left@primary","Spans: -"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":1},{"nodeIdx":0,"inputs":[],"core":{"title":"TableReader/1","details":["right@primary","Spans: -/1 and 1 other"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":0,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/4","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":0,"inputs":[],"core":{"title":"Aggregator/7","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":1,"inputs":[],"core":{"title":"TableReader/2","details":["right@primary","Spans: /1-/2"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":1,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/5","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":1,"inputs":[],"core":{"title":"Aggregator/8","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":2,"inputs":[],"core":{"title":"TableReader/3","details":["right@primary","Spans: /3-"]},"outputs":[{"title":"by hash","details":["@1"]}],"stage":2},{"nodeIdx":2,"inputs":[{"title":"unordered","details":[]},{"title":"ordered","details":["@1+"]}],"core":{"title":"MergeJoiner/6","details":["left(@1+)=right(@1+)","Out: None"]},"outputs":[],"stage":3},{"nodeIdx":2,"inputs":[],"core":{"title":"Aggregator/9","details":["COUNT_ROWS()"]},"outputs":[],"stage":4},{"nodeIdx":3,"inputs":[{"title":"unordered","details":[]}],"core":{"title":"Aggregator/10","details":["SUM_INT(@1)"]},"outputs":[],"stage":5},{"nodeIdx":3,"inputs":[],"core":{"title":"Response","details":[]},"outputs":[],"stage":0}],"edges":[{"sourceProc":0,"sourceOutput":1,"destProc":2,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":5,"destInput":1},{"sourceProc":0,"sourceOutput":1,"destProc":8,"destInput":1},{"sourceProc":1,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":1,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":2,"sourceOutput":0,"destProc":3,"destInput":0},{"sourceProc":3,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":4,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":4,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":5,"sourceOutput":0,"destProc":6,"destInput":0},{"sourceProc":6,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":7,"sourceOutput":1,"destProc":2,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":5,"destInput":2},{"sourceProc":7,"sourceOutput":1,"destProc":8,"destInput":2},{"sourceProc":8,"sourceOutput":0,"destProc":9,"destInput":0},{"sourceProc":9,"sourceOutput":0,"destProc":10,"destInput":1},{"sourceProc":10,"sourceOutput":0,"destProc":11,"destInput":0}]}

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDistSQLRangeCachesIntegrationTest PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDistSQLRangeCachesIntegrationTest.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/distsql_physical_planner_test.go
+++ b/pkg/sql/distsql_physical_planner_test.go
@@ -42,7 +42,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
@@ -337,8 +336,6 @@ func TestDistSQLReceiverUpdatesCaches(t *testing.T) {
 func TestDistSQLRangeCachesIntegrationTest(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-
-	skip.WithIssue(t, 52682)
 
 	// We're going to setup a cluster with 4 nodes. The last one will not be a
 	// target of any replication so that its caches stay virgin.

